soc/arm/nxp_imx/mimx8ml8_m7/Kconfig.defconfig.mimx8ml8_m7

```diff
@@ -34,6 +34,10 @@ config UART_MCUX_IUART
 
 endif # SERIAL
 
+config IPM_IMX_REV2
+	default y
+	depends on IPM
+
```

Comment:
Maybe move this to a separate commit, also shouldn't the `compatible = "nxp,imx-mu";` entries be added to `nxp_imx8ml_m7.dtsi` to add the actual devices?